cfbs/commands.py

```diff
@@ -401,19 +402,47 @@ def _someone_needs_me(this) -> bool:
     return 0
 
 
-@commit_after_command("Updated all modules")
-def update_command():
-    new_deps = []
-    new_deps_added_by = dict()
+def update_command(to_update):
     config = CFBSConfig.get_instance()
     index = config.index
+
+    if to_update:
+        to_update = [Module(m) for m in to_update]
+        index.translate_aliases(to_update)
+        skip = (
+            m for m in to_update if all(n["name"] != m.name for n in config["build"])
```

Comment:
I'd use `not any(...==..)` because it reads better to me than `all(...!=...)`, but YMMV.

Reply:
Not really a strong preference, but I think I prefer `all`